AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -43,16 +40,57 @@ export var functionsFlow = {
 
         if (!await isKuduAccessiblePromise)
         {
-            if (siteInfo.kind.includes("linux") || siteInfo.kind.includes("container")){
-                flowMgr.addView(new CommonWordings().connectivityCheckUnsupported.get());
-            }else{
-                flowMgr.addView(new VnetDnsWordings().cannotCheckWithoutKudu.get("Functions settings"));
-            }
+            flowMgr.addView(new VnetDnsWordings().cannotCheckWithoutKudu.get("Functions settings"));
             return;
         }
-
-        var isDaasExtAccessible = await diagProvider.checkDaasExtReachable(30);
-
+        var param = [];
+        param.push("api-version=2015-08-01");
+        var daasVersionInfo=await diagProvider.getDaaSExtApiAsync("daasversion",param); 
+        if(daasVersionInfo.status == "200")
+        {
+            var isDaasExtAccessible= true;
+        }
+        else
+        {
+            var isDaasExtAccessible= false;

```

Comment:
This logic seems wrong.  the "daasversion" api will not be implemented in older versions of daas extension but this does not mean the older version daas extension is not accessible.

Reply:
This is definitely a bug.  Have you tested this with the current daas extension version that does not have a "daasversion" api?

Reply:
Yes sid, I have tested this one and working as expected. 
The "daasversion" api was not implemented from our end.

Reply:
not a bug